File: framework_lib/src/main/java/com/network/bean/AreaBean.java
package com.network.bean;

import com.bean.Entity;

import java.util.List;

/**
 * @author yangsk
 * @class describe  {@link #}
 * @time 06/08/2018 3:01 PM
 */

public class AreaBean extends Entity {

    public List<ProvincesBean> provinces;
    public List<CitiesBean> cities;
    public List<CountiesBean> counties;

    public static class ProvincesBean {
        public int id;
        public String name;

    }

    public static class CitiesBean {
        public int id;
        public String name;
        public String lat;
        public String lng;
        public int province_id;
    }

    public static class CountiesBean {
        public int id;
        public String name;
        public int city_id;
        public String area_code;
        public String postcode;
        public String lat;
        public String lng;
    }
}

File: framework_lib/src/main/java/com/network/bean/AreaBean.kt
package com.network.bean

import com.bean.Entity

/**
 * @author yangsk
 * @class describe  [.]
 * @time 06/08/2018 3:01 PM
 */

class AreaBean : Entity() {

    var provinces: List<ProvincesBean>? = null
    var cities: List<CitiesBean>? = null
    var counties: List<CountiesBean>? = null

    class ProvincesBean {
        var id: Int = 0
        var name: String? = null

    }

    class CitiesBean {
        var id: Int = 0
        var name: String? = null
        var lat: String? = null
        var lng: String? = null
        var province_id: Int = 0
    }

    class CountiesBean {
        var id: Int = 0
        var name: String? = null
        var city_id: Int = 0
        var area_code: String? = null
        var postcode: String? = null
        var lat: String? = null
        var lng: String? = null
    }
}
